.github/workflows/android-prod.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           VITE_RGS: https://scorer.mutinywallet.com/v1/rgs/snapshot/
           VITE_AUTH: https://auth.mutinywallet.com
           VITE_SUBSCRIPTIONS: https://subscriptions.mutinywallet.com
-          VITE_STORAGE: https://storage.mutinywallet.com
+          VITE_STORAGE: https://storage.mutinywallet.com/v2
```

Comment:
There's many more places this needs to also be updated.

Reply:
Ah, i missed the staging urls

Reply:
Go ahead and apply to cloudflare now so we don't forget. it won't apply until a release, though it may break staging stuff in other PR's until then, but we need to test it in this branch.